templates/java/org/prism/Nodes.java.erb

```diff
@@ -4,6 +4,10 @@ package org.prism;
 import java.lang.Override;
 import java.lang.String;
 import java.lang.StringBuilder;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
```

Comment:
I'm assuming all of these hit the minimum required Java for JRuby/TruffleRuby?

Reply:
That's checked by the `build-java` CI job which uses Java 1.8 (specified in the Rakefile).